### PR TITLE
Array mirror inverse - C

### PR DESCRIPTION
**Pull request by SangeetaMishra**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#661

### CODE REVIEW

File: C/cp/Array_Mirror_Inverse.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+
```
Comment (iamrajiv) [inline C/cp/Array_Mirror_Inverse.c:3]: @SangeetaMishra To use `bool` in C you have to import `stdbool.h`.

File: C/cp/Array_Mirror_Inverse.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+int main() {
+  int n;
+  scanf("%d", &n);
+  int arr[n];
+  for (int i = 0; i < n; ++i) {
+  	scanf("%d", &arr[i]);
+  }
+  bool f = 1;
```
Comment (iamrajiv) [inline C/cp/Array_Mirror_Inverse.c]: @SangeetaMishra With `bool` `true` and `false` looks good rather `1` and `0`.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]
LGTM :)

**Comment by SangeetaMishra** [comment]
@Abhijit2505 Can you please review the PR

**Review by siddharth25pandey — APPROVED** [review]
@SangeetaMishra Good to go
Able to merge
Thanks!

**Pull request merged**